examples/src/test/scala/docs/guide/EventGeneratorApp.scala

```diff
@@ -28,59 +35,60 @@ object EventGeneratorApp extends App {
   final case object Start extends Command
 
   val shoppingCartsTag = "shopping-cart"
-  val config = ConfigFactory.parseResources("guide-shopping-cart-app.conf")
+  val config = ConfigFactory
+    .parseString("""
+      |akka.actor.provider = "cluster"
```

Comment:
Minor detail (this code is not displayed on the docs).

I see the shopping cart is sharded so we need clustering. But do we need the shopping cart to be sharded?

Reply:
I think it should be sharded because that's our recommendation for whenever `EventSourcedBehavior` is in use. 

We can document the `EventGeneratorApp` as we finalize the Getting Started Guide